asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/PriorRandom.js

```diff
@@ -132,6 +150,41 @@ const PriorRandom = (props) => {
                 <ArrowBack />
               </IconButton>
             </Tooltip>
+            <Stack className={classes.select} direction="row" spacing={1}>
+              <Typography sx={{ color: "text.secondary" }}>Show</Typography>
+              <FormControl variant="standard" sx={{ width: "48px" }}>
+                <Select value={nRecords} onChange={handleNRecordsChange}>
+                  <MenuItem value={5}>5</MenuItem>
```

Comment:
Can we start with 1?

Reply:
Yes, this would be nice. 1-10 with 5 as a default.